build.gradle

```diff
@@ -376,3 +382,7 @@ buildScan {
   licenseAgreementUrl = 'https://gradle.com/terms-of-service'
   licenseAgree = 'yes'
 }
+
+subprojects {
+    task allDepInsight(type: DependencyInsightReportTask) << {}
```

Comment:
I left it accidentally, but that task is very useful to check where/if there is an old asm (or anything else) version defined in any subproject. So, I propose to keep it.
More information: https://solidsoft.wordpress.com/2014/11/19/gradle-tricks-tracking-down-not-expected-transitive-dependencies/